app/views/admin/tags/show.html.erb

```diff
@@ -11,84 +11,35 @@
-    <% end %>
-    <%= form_for [:admin, @tag], html: { class: "form-inline justify-content-between" } do |f| %>
-      <div class="form-group">
-        <%= f.label :tag_moderator_id, "Add Moderator (id):", class: "mr-3" %>
-        <%= f.text_field :tag_moderator_id, class: "form-control" %>
-      </div>
-      <div>
-        <%= f.submit "Add Moderator", class: "btn btn-primary" %>
-      </div>
-    <% end %>
-  </div>
-
-  <div class="crayons-card p-6">
-    <h4 class="mb-4">Edit details</h4>
-    <%= form_for [:admin, @tag] do |f| %>
-      <div class="form-group">
-        <%= f.label :supported %>
-        <%= f.check_box :supported %>
-      </div>
-      <div class="form-group">
-        <%= f.label :badge_id %>
-        <% badges_array = Badge.pluck(:title, :id) %>
-        <%= f.select(:badge_id, options_for_select([["None", nil]] + badges_array, @tag.badge_id)) %>
-      </div>
-      <div class="form-group">
-        <%= f.label :category %>
-        <%= f.select(:category, options_for_select(Tag.valid_categories, @tag.category)) %>
-      </div>
-      <div class="form-group">
-        <%= f.label :social_preview_template %>
-        <%= f.select(:social_preview_template, Tag.social_preview_templates) %>
-      </div>
-      <div class="form-group">
-        <%= f.label :alias_for %>
-        <%= f.text_field :alias_for, class: "form-control" %>
-      </div>
-      <div class="form-group">
-        <%= f.label :pretty_name %>
-        <%= f.text_field :pretty_name, class: "form-control" %>
-      </div>
-      <div class="form-group">
-        <%= f.label :short_summary %>
-        <%= f.text_field :short_summary, class: "form-control" %>
-      </div>
-      <div class="form-group">
-        <%= f.label :rules_markdown %>
-        <br>
-        <%= f.text_area :rules_markdown, class: "form-control" %>
-      </div>
-      <div class="form-group">
-        <%= f.label :wiki_body_markdown %>
-        <br>
-        <%= f.text_area :wiki_body_markdown, class: "form-control" %>
-      </div>
-      <div class="form-group">
-        <%= f.label :bg_color_hex %>
-        <%= f.color_field :bg_color_hex, class: "form-control", required: true %>
-      </div>
-      <div class="form-group">
-        <%= f.label :text_color_hex %>
-        <%= f.color_field :text_color_hex, class: "form-control", required: true, value: @tag.text_color_hex.presence || "#ffffff" %>
-      </div>
-      <%= f.submit class: "btn btn-primary float-right" %>
-    <% end %>
+    <div class="crayons-card__header">
+      <h4 class="crayons-card__headline">Moderators</h4>
+    </div>
+    <div class="crayons-card__body flex flex-col gap-2">
+      <% if @tag.tag_moderator_ids.any? %>
+        <ul class="list-none">
+          <% @tag.tag_moderator_ids.each do |id| %>
+              <li class="flex justify-between">
+                <a href="/<%= User.find_by(id: id)&.username %>">@<%= User.find_by(id: id)&.username %></a>
```

Comment:
I think this is a bit too much AR for a view. Also this should probably use `URL.user` instead of manually constructing the link.